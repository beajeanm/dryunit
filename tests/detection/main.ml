let () =
  Printf.printf "%s\n"
    [%dryunit_debug]
