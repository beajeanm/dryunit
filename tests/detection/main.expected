First thing
Second thing
Test3
In secondary
